apps/src/templates/certificates/SuggestedAssignableCourses.jsx

```diff
@@ -0,0 +1,139 @@
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+
+import Alert from '@cdo/apps/componentLibrary/alert/Alert';
+import {BodyTwoText, Heading2} from '@cdo/apps/componentLibrary/typography';
+import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
+import i18n from '@cdo/locale';
+
+import style from './suggested_assignable_courses.module.scss';
+
+function SuggestedAssignableCourses({assignableCourseSuggestions, isEnglish}) {
+  const [expandedCardKey, setExpandedCardKey] = useState(null);
+  const handleQuickViewClicked = key => {
+    if (expandedCardKey === key) {
+      setExpandedCardKey(null);
+    } else {
+      setExpandedCardKey(key);
+    }
+  };
+
+  const [assignSuccess, setAssignSuccess] = useState(false);
+
+  useEffect(() => {
+    let timeoutId;
+    if (assignSuccess) {
+      timeoutId = setTimeout(() => {
+        setAssignSuccess(false);
+      }, 7000);
```

Comment:
What's the purpose of the timeout here?

Reply:
To make the alert disappear after some time. You can see it in the assign to section video in the description. Definitely open to feedback on whether this is necessary and whether the time should be adjusted!

Reply:
Oh gotcha, no this is perfect just wanted to understand!